src/main/java/io/github/codenilson/lavava2025/entities/dto/match/MatchCreateDTO.java

```diff
@@ -10,6 +10,8 @@
 @Data
 public class MatchCreateDTO {
 
```

Comment:
Creio que deveria ser obrigatório passar o mapa que a partida aconteceu/acontecerá. Adicionando essa anotação (funciona somente em `String`), só será aceito a requisição que passar um valor para `map`. O que acha?

```suggestion
    @NotBlank
```